claragenomics/dl4atac/utils.py

```diff
@@ -234,4 +232,4 @@ def log(self, names, normalizer=1.0, reset=True):
             elapsed_time = self.timers[name].elapsed(
                 reset=reset) * 1000.0/ normalizer
             string += ' | {}: {:.2f}'.format(name, elapsed_time)
-        print_rank_0(string)
+        print(string)
```

Comment:
I think `print_rank_0` forces prints to be flushed to the process associated with GPU 0, if I'm not mistaken. This helps keep print statements somewhat organized. Any particular reason for going to `print`?

Reply:
I didn't find print_rank_0 function anywhere. To make it functional, I changed it to print.

Reply:
moving this to an issue.